packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -89,7 +89,7 @@ const networks: Networks = {
     partnerChainID: '1',
     isArbitrum: true,
     tokenBridge: mainnetBridge,
-    confirmPeriodBlocks: 48384,
+    confirmPeriodBlocks: 45818,
```

Comment:
I think we could remove this from here.
It isn't being used anywhere anymore, we can add it back in if needed